meteoritics-and-planetary-science.csl

```diff
@@ -112,7 +112,7 @@
     <text term="in" text-case="capitalize-first" suffix=" "/>
     <text variable="container-title" font-style="italic"/>
   </macro>
-  <citation et-al-min="10" et-al-use-first="1" et-al-subsequent-min="6" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">
+  <citation et-al-min="2" et-al-use-first="1" et-al-subsequent-min="2" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">
```

Comment:
Please do:
`<citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">`
I realize they say "two or more" in their instructions, but that's not what they mean (check the journal -- in text citation of two authors are (firstauthor and secondauthor year). The subsequent bit doesn't do anything.